climpred/constants.py

```diff
@@ -2,6 +2,16 @@
 CLIMPRED_ENSEMBLE_DIMS = ['init', 'member', 'lead']
 CLIMPRED_DIMS = CLIMPRED_ENSEMBLE_DIMS + ['time']
 
+# Valid units for lead dimension
+VALID_LEAD_UNITS = ['years', 'seasons', 'months', 'weeks', 'pentads', 'days']
+
+# List of frequencies to check a dimension has different coords
```

Comment:
I would say "List of frequencies to check to infer time series stride."

Reply:
And maybe change the name to `FREQ_LIST_TO_INFER_STRIDE`. I know it's verbose, but `FREQ_LIST` is only used in a specific instance and the name currently is vague.